README.md

```diff
@@ -119,23 +119,57 @@ cargo build -p millau-bridge-node
 cargo build -p substrate-relay
 ```
 
-### Running
+### Running dev network
+
+We will only launch a dev network to demonstrate the unidirectional relay from millau to rialto here
+
+#### Running from local script
 
 To run a simple dev network you'll can use the scripts located in
 [the `deployments/local-scripts` folder](./deployments/local-scripts). Since the relayer connects to
-both Substrate chains it must be run last.
+both Substrate chains it must be run last. 
 
 ```bash
-# In `parity-bridges-common` folder
+
+# run rialto node
 ./deployments/local-scripts/run-rialto-node.sh
+
+# run millau node
 ./deployments/local-scripts/run-millau-node.sh
+
+# init bridge & relay headers from millau to rialto
 ./deployments/local-scripts/relay-millau-to-rialto.sh
+
+# relay messages from millau to rialto
+./deployments/local-scripts/relay-messages-millau-to-rialto.sh
```

Comment:
We also need to run the other side of the message bridge (Rialto -> Millau) or else message acknowledgements won't work